cmd/guacone/cmd/osv.go

```diff
@@ -191,7 +192,7 @@ var osvCmd = &cobra.Command{
 		wg.Wait()
 		cf()
 
-		if gotErr {
+		if gotErr == 1 {
```

Comment:
This has to use the `atomic` package to compare. To avoid the race. https://pkg.go.dev/sync/atomic#LoadInt32